content/en/docs/components/pipelines/reference/component-spec.md

```diff
@@ -1,7 +1,7 @@
 +++
 title = "Component Specification"
 description = "Definition of a Kubeflow Pipelines component"
-weight = 10
+weight = 4
                     
 +++
 {{% alert title="Out of date" color="warning" %}}
```

Comment:
We should remove this warning, as it's not correct and has been on this page forever.

There are also a number of these warnings in the new `Legacy (V1)` section, which are all incorrect and can also be removed to avoid confusing users.

Reply:
> We should remove this warning, as it's not correct and has been on this page forever.
Done.

> There are also a number of these warnings in the new Legacy (V1) section, which are all incorrect and can also be removed to avoid confusing users.
I opened a [new issue](https://github.com/kubeflow/website/issues/3751) for that. If possible, can you point to the pages (if not all of them) that can have the warning removed?

Reply:
@hbelmiro It's literally all of the pages under the pipelines section which contain `{{% alert title="Out of date" color="warning" %}}`, there are currently 5 (in master):

- Component Specification
     - You removed this one already.
- Best Practices for Designing Components
- DSL Recursion
- Using environment variables in pipelines
- Manipulate Kubernetes Resources as Part of a Pipeline
    - Looking at this one,  we also need to create a V2 version of this, because its changed a lot.
    - See https://github.com/kubeflow/website/issues/3750#issuecomment-2148396967